### PR TITLE
Bring master and dev to same state

### PR DESCRIPTION
**Pull request by NirantK**

### CODE REVIEW

**Comment by NirantK** [comment]
Don't push to `main` directly! 

Follow the spirit of this please? https://trunkbaseddevelopment.com/

Also, @meghanabhange -- please review this ASAP?

File: .github/workflows/secrets.yaml [hunk 1]
```diff
@@ -1,41 +1,24 @@
 name: twitter bot
-
 on: 
-
   push:
-
     branches:
-      - streamlit-app
+      - main
```
Comment (meghanabhange) [inline .github/workflows/secrets.yaml:5]: Should we add dev here too?

File: .github/workflows/secrets.yaml [hunk 2]
```diff
@@ -1,41 +1,24 @@
...
+        with: 
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: hinglish-nlp
+          heroku_email: radhika.sethi0312@gmail.com
         env:
-            
-          ACCESS_KEY: ${{ secrets.ACCESS_KEY }}
-          ACCESS_SECRET: ${{ secrets.ACCESS_SECRET }}
-          CONSUMER_KEY: ${{ secrets.CONSUMER_KEY }}
-          CONSUMER_SECRET: ${{ secrets.CONSUMER_SECRET }}
```
Comment (meghanabhange) [inline .github/workflows/secrets.yaml:38]: Don't we need to keep these access keys as well? Like keeping with Access key and secret access keys. 
Also should we comment out the heroku workflow for now given that we have deployed it on Digital Oceans App platform?

**Review by meghanabhange — CHANGES_REQUESTED** [review]

File: .github/workflows/secrets.yaml (hunk 1 above)
Comment (NirantK) [reply .github/workflows/secrets.yaml:5]: Sounds like a good idea. But after merge I think. Push to dev and then raise a PR to master

File: .github/workflows/secrets.yaml (hunk 2 above)
Comment (NirantK) [reply .github/workflows/secrets.yaml:38]: Yes, please remove/comment out the Heroku workflow.

**Comment by meghanabhange** [comment]
Merging this to get the branches upto date. Then will resolve the above issues in a separate branch and raise PR to dev and master.

**Pull request merged**